query-algebrizer/src/cc.rs

```diff
@@ -83,6 +87,12 @@ impl<T> OptionEffect<T> for Option<T> {
     }
 }
 
+fn unit_type_set(t: ValueType) -> HashSet<ValueType> {
```

Comment:
You can do `vec![1, 2].into_iter().collect()` with the right type bounds, but it's messy.